components/script/dom/subtlecrypto.rs

```diff
@@ -0,0 +1,250 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use std::rc::Rc;
+
+use aes::cipher::KeyInit;
+use aes::{Aes128, Aes192, Aes256};
```

Comment:
I'm curious about the use of `aes`. Is it possible to use a higher-level library like `ring`?

Reply:
Potentially, according to some prior investigation by jdm though ring doesn't have support for AES-CBC https://github.com/servo/servo/issues/26878#issuecomment-658241844